user.rb

```diff
@@ -0,0 +1,17 @@
+class User
+  attr_accessor :name, :pin, :balance
+
+  def initialize(name, pin, balance)
+    @name    = name
+    @pin     = pin
+    @balance = balance.to_i
+  end
+
+  def check_balance
+    "#{name}, your current balance is #{balance}."
+  end
+
+  def withdraw
+    @balance - withdraw_amount
```

Comment:
I don't see anywhere in the code where this method is actually called. Is this leftover code?